### ISSUE TITLE
NavigationBar combo showing "<Global Scope>" when typing inside class code

### ISSUE THREAD

**Issue by cpyrgas** [X-Sharp/XSharpPublic#1039]
This lloks quite annoying, I think it needs to be fixed for 2.12. Paste the following code in the editor (I think the existance of the function at the end is the only thing really needed to reproduce this), and type anything inside the constructor code. The left hand side navigation combo now always changes and displays `"<Global Scope>"`

```
CLASS HelpAbout 

CONSTRUCTOR( oOwnerApp, oMenu, oToolbar ) 
LOCAL n AS INT
n := 112
n := 1 // comment here

? "assd"

n := 2
END CLASS


FUNCTION SomeFuncAtTheEnd() AS LOGIC
RETURN TRUE
```

**Comment by RobertvanderHulst** [comment]
Chris,
It is time to wrap up 2.12.
"Annoying" is not a show stopper.

**Comment by RobertvanderHulst** [comment]
Fortunately this one was easy to fix, so I''ll include it in 2.12.
